package.json

```diff
@@ -18,10 +18,10 @@
     "test": "jest",
     "test:ci": "jest --runInBand",
     "security_audit": "npx audit-ci --config audit-ci.json",
-    "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open --e2e --browser chrome",
-    "test:integration": "npm run start-test-wiremock && npx start-server-and-test start-feature http://localhost:3007/ping int-test",
-    "test:integration:ui": "npm run start-test-wiremock && npx start-server-and-test start-feature:dev http://localhost:3007/ping int-test-ui",
+    "test:integration:ci": "cypress run --config video=false",
+    "test:integration:ui:ci": "cypress open --e2e --browser chrome",
+    "test:integration:local": "npm run start-test-wiremock && npx start-server-and-test start-feature http://localhost:3007/ping test:integration",
```

Comment:
We'll need to update the local scripts to use the new `:ci` naming too.

Reply:
aha yes :)